test/extended/images/mongodb_ephemeral.go

```diff
@@ -0,0 +1,64 @@
+package images
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+	"time"
+)
+
+var _ = g.Describe("images: mongodb: ephemeral template", func() {
+	defer g.GinkgoRecover()
+	var (
+		templatePath = exutil.FixturePath("..", "..", "examples", "db-templates", "mongodb-ephemeral-template.json")
+		oc           = exutil.NewCLI("mongodb-create", exutil.KubeConfigPath()).Verbose()
+	)
+
+	g.Describe("Creating from a template", func() {
+		g.It(fmt.Sprintf("should process and create the %q template", templatePath), func() {
+			oc.SetOutputDir(exutil.TestContext.OutputDir)
+
+			g.By(fmt.Sprintf("calling oc process -f %q", templatePath))
+			configFile, err := oc.Run("process").Args("-f", templatePath).OutputToFile("config.json")
+			o.Expect(err).Should(o.Succeed())
+
+			g.By(fmt.Sprintf("calling oc create -f %q", configFile))
+			o.Expect(oc.Run("create").Args("-f", configFile).Execute()).Should(o.Succeed())
+
+			g.By("expecting the mongodb service get endpoints")
+			o.Expect(oc.KubeFramework().WaitForAnEndpoint("mongodb")).Should(o.Succeed())
+
+			g.By("expecting the mongodb service answers for ping")
+
+			podNames, err := exutil.WaitForPods(
+				oc.KubeREST().Pods(oc.Namespace()),
+				exutil.ParseLabelsOrDie("name=mongodb"),
+				exutil.CheckPodIsRunningFn,
+				1,
+				time.Minute,
+			)
+			o.Expect(err).Should(o.Succeed())
+
+			mongoPod := podNames[0]
```

Comment:
Is it possible that `podNames` is empty? Check that there's something there before accessing the 0-th index? Do we want to check that there's only one pod name in the result?